js/cpu6502.ts

```diff
@@ -76,6 +76,16 @@ const sizes: Modes = {
 /** Status register flag numbers. */
 type flag = 0x80 | 0x40 | 0x20 | 0x10 | 0x08 | 0x04 | 0x02 | 0x01;
 
+export type DebugInfo = [
```

Comment:
Should this be an object instead? That would allow access like `info.pc`, which now has to be `info[0]`. See `debugger.ts` line 20.